tests/lit-tests/func_template_nontype_10.ispc

```diff
@@ -0,0 +1,33 @@
+// Instantiation of templates with bool parameters.
+// RUN: %{ispc}  %s --emit-llvm-text --target=host --nostdlib -o - | FileCheck %s
+
+// CHECK-LABEL: define {{.*}} @somecode_with_fixed_path1
+// CHECK: call <[[WIDTH:.*]] x [[TYPE:.*]]> @somecode___Cunbtrue
+
+// CHECK-LABEL: define {{.*}} @somecode___Cunbtrue
+// CHECK: call <[[WIDTH:.*]] x [[TYPE:.*]]> @true_call___
```

Comment:
Probably do a CHECK-NOT for the other branch?

Reply:
I slightly changed a test to validate nested templates as well.

Reply:
I we check that one branch is there, we assume that the other one should not be there, right? Otherwise, if both are present, the check seemed to be pointless, right?

Reply:
Fixed